modules/persistent-storage-csi-snapshots-restore.adoc

```diff
@@ -7,11 +7,13 @@
 
 After your `VolumeSnapshot` object is bound, you can use that object to provision a new volume that is pre-populated with data from the snapshot.
 
+When `volumemsnapshot.status.readyToUse` = `true`, the restore is successful and a new persistent volume claim (PVC) is created.
```

Comment:
[BZ1906281](https://bugzilla.redhat.com/show_bug.cgi?id=1906281)
@qinpingli - with this `true` statement, is it still helpful to include the next sentence, or should we remove it?
> The volume snapshot content object is used to restore the existing volume to a previous state.

Reply:
@bobfuru Here is not correct, **When `volumemsnapshot.status.readyToUse` = `true`** is another condition that **you can use that object to provision a new volume that is pre-populated with data from the snapshot.**, this doesn't mean **the restore is successful and a new persistent volume claim (PVC) is created.**
cc @qinpingli

Reply:
I revised to this statement to state:

> After your `VolumeSnapshot` object is bound and the `readyToUse` value is set to `true`, you can use that object to provision a new volume that is pre-populated with data from the snapshot.